src/components/Layout/DrawerNavigation.tsx

```diff
@@ -0,0 +1,196 @@
+
+export const DrawerNavigationComponent: FunctionComponent<DrawerContentComponentProps> = ({
+  navigation
+}) => {
+  const { logout } = useLogout();
+  const showHelpModal = useContext(HelpModalContext);
+  const { drawerButtons } = useDrawerContext();
+  const handleLogout = useCallback((): void => {
+    logout(navigation.dispatch);
+  }, [logout, navigation.dispatch]);
+
+  const onPressLogout = (): void => {
+    Alert.alert(
+      "Are you sure you want to log out?",
+      "Stay logged in to change your scanning location.",
+      [
+        {
+          text: "Cancel"
+        },
+        {
+          text: "Logout",
+          onPress: handleLogout,
+          style: "destructive"
+        }
+      ],
+      { cancelable: false }
+    );
+  };
+
+  const onPressCloseDrawer = (): void => {
+    navigation.dispatch(DrawerActions.closeDrawer());
+  };
+
+  return (
+    <SafeAreaView
+      style={styles.container}
+      forceInset={{ top: "always", horizontal: "never" }}
+    >
+      <TouchableOpacity
+        onPress={onPressCloseDrawer}
+        style={{
+          position: "absolute",
+          right: size(3),
+          top: size(5),
+          padding: size(1)
+        }}
+      >
+        <MaterialCommunityIcons
+          name="close"
+          size={size(3)}
+          color={color("blue", 50)}
+        />
+      </TouchableOpacity>
+      <View
+        style={{
+          marginTop: size(9),
+          borderTopColor: color("grey", 20),
+          borderTopWidth: 1
+        }}
+      >
+        {drawerButtons.map(button => (
+          <DrawerButton {...button} key={button.label} />
+        ))}
+        <TouchableOpacity onPress={onPressLogout}>
+          <View style={styles.navLink}>
+            <MaterialCommunityIcons
+              name="logout"
+              size={size(2.5)}
+              color={color("blue", 50)}
+              style={{ marginRight: size(2) }}
+            />
+            <AppText style={styles.navLinkText}>Logout</AppText>
+          </View>
+        </TouchableOpacity>
+      </View>
+      <View style={{ marginTop: "auto", marginBottom: size(4) }}>
+        <BottomNavigationLink onPress={showHelpModal}>
+          Help & Support
+        </BottomNavigationLink>
+        <BottomNavigationLink
+          onPress={() => {
+            Linking.openURL("https://www.supplyally.gov.sg/terms-of-use");
+          }}
+        >
+          Terms of use
+        </BottomNavigationLink>
+        <BottomNavigationLink
+          onPress={() => {
+            Linking.openURL("https://www.supplyally.gov.sg/privacy");
+          }}
+        >
+          Privacy Statement
+        </BottomNavigationLink>
+        <BottomNavigationLink
+          onPress={() => {
+            Linking.openURL("https://www.tech.gov.sg/report_vulnerability");
+          }}
+        >
+          Report vulnerability
+        </BottomNavigationLink>
+        <View style={{ marginTop: size(3), ...styles.bottomNavContainerLink }}>
```

Comment:
Can use a different style "class" instead, it isn't a "bottomNavContainerLink", also we don't need the view since we can style the AppText instead